CMakeLists.txt

```diff
@@ -9,6 +9,21 @@ project(${PROJECT_NAME})
 
 set(CMAKE_CXX_FLAGS "-g -Wall")
 
+# force static runtime libraries for msvc builds
```

Comment:
Thanks for the PR - can you elaborate on this comment more, and clarify what MSVC is?